core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -207,10 +207,10 @@
         <source>The class(es) will be used in the navigation menu and the &amp;lt;body&amp;gt; tag.</source>
       </trans-unit>
       <trans-unit id="tl_page.sitemap.0">
-        <source>Show in sitemap</source>
+        <source>Show in HTML sitemap</source>
       </trans-unit>
       <trans-unit id="tl_page.sitemap.1">
-        <source>Here you can define whether the page is shown in the sitemap.</source>
+        <source>Here you can define whether the page is shown in the HTML sitemap.</source>
```

Comment:
```suggestion
        <source>Here you can define whether the page is shown in the front end module sitemap.</source>
```

Reply:
We agreed on the term `HTML sitemap` in the last mumble call as this is what SEO people are apparently calling it.

Reply:
That is the title, but it would be helpful if the explanation contains additional information.